topics/MBC_cluster_conversion/mbc_divider_script.py

```diff
@@ -240,6 +243,17 @@ def wait_for_container_to_be_ready(container_type, sudo):
         if local_status[container_type.container_name()]['status']['internalStatus']['state'] == 'READY':
             break
 
+def wait_for_host_deactivate(host_id):
+    get_host_from_hosts_list_cmd = '/bin/sh -c "weka cluster host {} -J"'.format(host_id)
+    logger.info('Waiting for host to deactivate')
+    retries = 60
+    for i in range(retries):
+        host_row = json.loads(run_shell_command(get_host_from_hosts_list_cmd))
+        if host_row[0]['status'] == 'INACTIVE':
```

Comment:
not quite safe..
maybe something like
```
if 'status'  in host_row[0] and host_row['status'] == 'INACTIVE':
```
or
```
status = host_row.get('status', '')
if status == 'INACTIVE':
```
maybe add some IndexError/ json.JSONDecodeError as well